webpack.config.js

```diff
@@ -444,6 +444,19 @@ module.exports = {
                     },
                 ]
             },
+            {
+                // See https://github.com/patternfly/patternfly-react/issues/3815 and
+                // [Redefine grid breakpoints section in pkg/lib/_global-variables.scss for more details)
```

Comment:
Supposedly this is supposed to read "Redefine grid breakpoints" section.. ?
It's not clear how this magically uses that file here.

Reply:
The file is made visible to all scss/css files here https://github.com/cockpit-project/cockpit/blob/master/webpack.config.js#L439

Anyway - I seperated this to another PR because it's rather complicated change that can be standalone. https://github.com/cockpit-project/cockpit/pull/13807

I have the explanation in the commit message but if it's still not clear I am happy to explain in more detail.